xcode/fast_obj/fast_obj.xcodeproj/project.pbxproj

```diff
@@ -0,0 +1,191 @@
+// !$*UTF8*$!
```

Comment:
As it was mentioned before, can we avoid creating a separate library at all, and just include the header where necessary?

Reply:
Up to you